static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -61,10 +62,7 @@ function GroupRelatedIssues({params}: Props) {
       if (item.type === 'trace_connected') {
         traceMeta = {...item.meta};
       }
-      // If the group we're looking related issues for shows up in the table,
-      // it will trigger a bug in getGroupReprocessingStatus because activites would be empty,
-      // thus, we excude it from the list of related issues
```

Comment:
We don't need to do this filtering since the endpoint does not return the current group.